File: google-ai-tutorials/2-Vertex_AI_Gemini_Pro_API_starter/vertex_gemini_pro.js
const { VertexAI } = require("@google-cloud/vertexai");

const projectId = "xxxxxxxxxxxxxxxxxxx";
const location = "us-central1";

const vertexAI = new VertexAI({ project: projectId, location: location });

async function run() {
  const model = vertexAI.preview.getGenerativeModel({ model: "gemini-pro" });
  const request = {
    contents: [{ role: "user", parts: [{ text: "Say hi" }] }],
  };
  const result = await model.generateContent(request);
  const response = await result.response;

  console.log(response.candidates[0].content.parts[0].text);
}

run();

File: google-tutorials/2-Vertex_AI_Gemini_Pro_API_starter/vertex_gemini_pro.py
import vertexai
from vertexai.preview.generative_models import GenerativeModel

project_id = 'xxxxxxxxxxxxxxxxxxx'
location = 'us-central1'

vertexai.init(project=project_id, location=location)

model = GenerativeModel('gemini-pro')
response = model.generate_content('Say hi')

print(response.text)